### ISSUE TITLE
:random module deprecated

### ISSUE THREAD

**Issue by OvermindDL1** [zackehh/cachex#48]
This happens while compiling cachex:
```elixir
==> cachex
Compiling 20 files (.ex)
warning: random:uniform/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/cachex/notifier.ex:46
```

**Comment by zackehh** [comment]
@OvermindDL1 is this OTP 19?

**Comment by OvermindDL1** [comment]
Correct, the `:random` module has been deprecated and will be removed in an up-coming version.

**Comment by zackehh** [comment]
Resolved by c435117.
